### PR TITLE
fix: Fix `ColorCircles` and `Sizes` initial render (no-changelog)

### PR DESCRIPTION
**Pull request by tomi**
## Summary

Fix `ColorCircles` and `Sizes` initial render

## Review / Merge checklist

- [x] PR title and summary are descriptive. ([conventions](../blob/master/.github/pull_request_title_conventions.md)) 
- [ ] [Docs updated](https://github.com/n8n-io/n8n-docs) or follow-up ticket created.
- [x] Tests included. 
- [ ] PR Labeled with `release/backport` (if the PR is an urgent fix that needs to be backported)

### CODE REVIEW

File: packages/design-system/src/styleguide/ColorCircles.vue [hunk 1]
```diff
@@ -42,8 +47,8 @@ onUnmounted(() => {
 });
 
 // Expose functions for template usage
-const getHexValue = (color: string) => getHex(hsl[color]);
-const getHSLValue = (color: string) => resolveHSLCalc(hsl[color]);
+const getHexValue = (color: string) => getHex(hsl.value[color]);
```
Comment (tomi) [inline packages/design-system/src/styleguide/ColorCircles.vue:50]: Since `setColors` was called in `onMounted`, `hsl` wasn't initialized yet

**Review by netroy — DISMISSED** [review]
🎉

**Review by netroy — APPROVED** [review]

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Comment by janober** [comment]
Got released with [`n8n@1.47.0`](https://github.com/n8n-io/n8n/releases/tag/n8n%401.47.0)

**Pull request merged**